common/scala/build.gradle

```diff
@@ -40,6 +40,8 @@ dependencies {
     compile 'com.google.code.findbugs:jsr305:3.0.2'
     compile 'io.kamon:kamon-core_2.11:0.6.7'
     compile 'io.kamon:kamon-statsd_2.11:0.6.7'
+    //for mesos
+    compile 'com.adobe.api.platform.runtime:mesos-actor:0.0.3'
```

Comment:
what is in mesos-actor (i guess i'll see below)... wondering if there are parts that should be in this repo perhaps. also if it's 0.0.3, what are the caveats about functionality, stability, etc?

Reply:
Not sure what parts or concerns there are specifically?